include/circt/Dialect/FIRRTL/FIRRTLTypesImpl.td

```diff
@@ -547,7 +487,7 @@ def BaseTypeAliasImpl : FIRRTLImplType<"BaseTypeAlias", [FieldIDTypeInterface],
 class PropImplType<string name,
                    list<Trait> traits = [],
                    string baseCppClass = "::circt::firrtl::FIRRTLType">
-  : FIRRTLImplType<name, traits # [ AtomicFieldIDTypeInterface ], baseCppClass>;
+  : FIRRTLImplType<name, traits # [ ], baseCppClass>;
```

Comment:
Like the others, 
```suggestion
  : FIRRTLImplType<name, traits, baseCppClass>;
```

Reply:
I did do that on purpose, but I can change it.